waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/KeyController.kt

```diff
@@ -270,6 +304,25 @@ fun Application.keys() = walletRoute {
                 val success = getWalletService().removeKey(keyId)
                 context.respond(if (success) HttpStatusCode.Accepted else HttpStatusCode.BadRequest)
             }
+
+            post("sign", {
+                summary = "Sign a message with a specific key"
+                request {
+                    body<JsonObject> { description = "The message to sign" }
```

Comment:
not sure about the implications and its worthiness
but this could be `JsonElement`
and this would support additionally json-string and json-array for signing